justfile

```diff
@@ -69,6 +69,9 @@ build config:
 build-firmware config firmware:
 	cargo run --package runner -- build -v --config {{config}} --firmware {{firmware}}
 
+miralis-firmware config:
+	cargo run --package runner -- build --config {{config}} --as-firmware true
+
```

Comment:
Do we need a new `just` rule or can we use the already existing `just build-firmware` with `miralis` as argument? It seems you changed the runner to recognize `miralis` as a special case, so we might not need the extra `just` rule :)

Reply:
You are right about that. I delete this extra command.